### PR TITLE
[standalone][ibm_vpc] avoid docker pull or load if runtime installed in the custom VM image

### PR DESCRIPTION
**Pull request by gilv**
…ged with runtime

 
Developer's Certificate of Origin 1.1

       By making a contribution to this project, I certify that:

       (a) The contribution was created in whole or in part by me and I
           have the right to submit it under the Apache License 2.0; or

       (b) The contribution is based upon previous work that, to the best
           of my knowledge, is covered under an appropriate open source
           license and I have the right under that license to submit that
           work with modifications, whether created in whole or in part
           by me, under the same open source license (unless I am
           permitted to submit under a different license), as indicated
           in the file; or

       (c) The contribution was provided directly to me by some other
           person who certified (a), (b) or (c) and I have not modified
           it.

       (d) I understand and agree that this project and the contribution
           are public and that a record of the contribution (including all
           personal information I submit with it, including my sign-off) is
           maintained indefinitely and may be redistributed consistent with
           this project or the open source license(s) involved.

### CODE REVIEW

**Comment by gilv** [comment]
@JosepSampe This PR about - if custom image already contains runtime image in the Docker, this prevents ``` docker pull <runtime> ``` and greatly improve times when many VM backends created and each need the runtime image.

File: lithops/localhost/localhost.py [hunk 1]
```diff
@@ -41,15 +41,16 @@ class LocalhostHandler:
     """
 
     def __init__(self, localhost_config):
-        logger.debug('Creating Localhost compute client')
+        logger.info('Creating Localhost compute client')
         self.config = localhost_config
         self.runtime = self.config['runtime']
+        self.local_runtime_load = self.config['local_runtime_load']
```
Comment (JosepSampe) [inline lithops/localhost/localhost.py:47]: @gilv I think LocalExecutor will fail due this. Better change it to: `self.local_runtime_load = self.config.get('local_runtime_load', False)`

**Comment by gilv** [comment]
@JosepSampe I just fixed the comment. Ok to merge?

**Comment by JosepSampe** [comment]
@gilv ok, go ahead ;)

File: lithops/localhost/localhost.py [hunk 2]
```diff
@@ -41,15 +41,16 @@ class LocalhostHandler:
     """
 
     def __init__(self, localhost_config):
-        logger.debug('Creating Localhost compute client')
+        logger.info('Creating Localhost compute client')
         self.config = localhost_config
         self.runtime = self.config['runtime']
+        self.local_runtime_load = self.config.get('local_runtime_load', False)
```
Comment (JosepSampe) [inline lithops/localhost/localhost.py:47]: @gilv I think that maybe this variable should be True by default

Comment (gilv) [reply lithops/localhost/localhost.py:47]: @JosepSampe it's a bit complicated process to create image + runtime. Not sure if majority will do it, even it greatly improve performance. When it false, Lithops will deploy runtime as always

**Pull request merged**